src/assets/HTMLAsset.js

```diff
@@ -67,7 +67,8 @@ const SCRIPT_TYPES = {
   'text/javascript': 'js',
   'application/json': false,
   'application/ld+json': 'jsonld',
-  'text/html': false
+  'text/html': false,
+  'text/x-template': 'html'
```

Comment:
Shouldn't this be just false if it's html? It's probably the same as `text/html` right?

Reply:
Should have looked at the issue first... Thanks for fixing this.